openapi3/internalize_refs.go

```diff
@@ -376,12 +379,12 @@ func (doc *T) derefPaths(paths map[string]*PathItem, refNameResolver RefNameReso
 // refNameResolver takes in references to returns a name to store the reference under locally.
 // It MUST return a unique name for each reference type.
 // A default implementation is provided that will suffice for most use cases. See the function
-// documention for more details.
+// documentation for more details.
 //
 // Example:
 //
-//	doc.InternalizeRefs(context.Background(), nil)
-func (doc *T) InternalizeRefs(ctx context.Context, refNameResolver func(ref string) string) {
```

Comment:
`Context` was not used anywhere in this function, so I removed it to avoid confusion when people try to use this function

Reply:
Please don't. That's an unrelated change involving API breakage. Let's leave this to a separate discussion.

Reply:
Fair enough, will revert